apps/chat-e2e/src/tests/duplicateConversation.test.ts

```diff
@@ -35,10 +27,10 @@ dialTest(
     const secondRequest = 'second request';
 
     await dialTest.step('Prepare conversation with some history', async () => {
-      conversation = conversationData.prepareModelConversationBasedOnRequests(
-        defaultModel,
```

Comment:
Could you tell what is the point of removing the defaultModel?

Reply:
The `model` param is optional for `prepareModelConversationBasedOnRequests()` function. If it is missing, the default model is used